jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -1036,6 +1042,9 @@ void jbmc_parse_optionst::help()
     " --property id                only check one specific property\n"
     " --stop-on-fail               stop analysis once a failed property is detected\n" // NOLINT(*)
     " --trace                      give a counterexample trace for failed properties\n" //NOLINT(*)
+    " --verify-trace               throw an error if the structure of the counterexample\n" // NOLINT(*)
+    "                              trace does not match certain assumptions\n"
+    "                              (experimental, currently java only)\n" //NOLINT(*)
```

Comment:
Ok, that's very helpful. But then shouldn't it be `validate-trace` just like we have `validate-ssa-equation` and `validate-goto-model`?

Reply:
That sounds like a fair suggestion. I had been flip-flopping anyway, so I've updated this option to be `validate-trace`